pkg/controller/catalog/storage/mem_storage_service_class.go

```diff
@@ -0,0 +1,47 @@
+package storage
+
+import (
+	"errors"
+
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
+)
+
+var (
+	errNoSuchServiceClass = errors.New("no such service class")
+)
+
+type memStorageServiceClass struct {
+	// This gets fetched when a SB is created (or possibly later when refetched).
+	// It's static for now to keep compatibility, seems like this could be more dynamic.
+	classes map[string]*servicecatalog.ServiceClass
+}
+
+func newMemStorageServiceClass() *memStorageServiceClass {
+	return &memStorageServiceClass{classes: make(map[string]*servicecatalog.ServiceClass)}
+}
+
+func (m *memStorageServiceClass) List() ([]*servicecatalog.ServiceClass, error) {
+	ret := make([]*servicecatalog.ServiceClass, len(m.classes))
+	i := 0
+	for _, val := range m.classes {
```

Comment:
@duglin I opened #200 to track this. I've also voluntarily assigned it to myself since I am happy to bit that off.